crates/turborepo-ui/src/tui/app.rs

```diff
@@ -160,6 +159,14 @@ fn poll(input_options: InputOptions, receiver: &AppReceiver, deadline: Instant)
     }
 }
 
+const MIN_HEIGHT: u16 = 10;
```

Comment:
These are completely arbitrary. Open to suggestions on what they should actually be

Reply:
I think these are completely reasonable. If I'm accounting for borders correctly 20x10 term would probably lead to a 12x8 terminal pane. Not sure if that's usable, but anything smaller is definitely not usable.

Reply:
add a comment that it's arbitrary?